gbe/views/act_tech_wizard_view.py

```diff
@@ -0,0 +1,277 @@
+from django.views.generic import View
+from django.views.decorators.cache import never_cache
+from django.utils.decorators import method_decorator
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+from django.utils.formats import date_format
+from django.core.management import call_command
+from django.core.urlresolvers import reverse
+from django.shortcuts import (
+    get_object_or_404,
+    render,
+)
+from gbe.functions import (
+    validate_perms,
+    validate_profile,
+)
+from gbe.forms import (
+    BasicActTechForm,
+    BasicRehearsalForm,
+)
+from gbe.models import (
+    Act,
+    GenericEvent,
+    Show,
+    UserMessage,
+)
+from gbetext import (
+    default_act_tech_basic_submit,
+    default_basic_acttech_instruct,
+    default_rehearsal_booked,
+    default_rehearsal_acttech_instruct,
+    rehearsal_book_error,
+)
+from scheduler.idd import (
+    get_occurrences,
+    get_schedule,
+    remove_booking,
+    set_act,
+)
+from gbe.scheduling.views.functions import show_general_status
+from scheduler.data_transfer import (
+    BookableAct,
+    ScheduleItem,
+)
+from django.contrib import messages
+from settings import GBE_DATETIME_FORMAT
+
+
+class ActTechWizardView(View):
+    template = 'gbe/act_tech_wizard.tmpl'
+    permissions = ('Tech Crew', )
+    default_event_type = None
+    page_title = 'Edit Act Technical Information'
+    first_title = 'Set Rehearsal Time'
+    second_title = 'Provide Technical Information'
+
+    def set_rehearsal_forms(self, request=None):
+        rehearsal_forms = []
+        possible_rehearsals = GenericEvent.objects.filter(
+            type='Rehearsal Slot',
+            e_conference=self.act.b_conference).values_list('eventitem_id')
+        for show in self.shows:
+            response = get_occurrences(
+                labels=[self.act.b_conference.conference_slug],
+                foreign_event_ids=possible_rehearsals,
+                parent_event_id=show.pk)
+            choices = []
+            initial = None
+
+            for event in response.occurrences:
```

Comment:
with this algorithm, if the data is ever weird (2 rehearsals, 1 act, 1 show), the outcome of editing the rehearsal form will NOT correct it.
I don't care enough to add complexity, though.